lib/debug/thread_client.rb

```diff
@@ -340,9 +340,9 @@ def instance_eval_for_cmethod frame_self, src
       frame_self.instance_eval(src)
     end
 
-    SPECIAL_LOCALS = [
-      [:raised_exception, "_raised_"],
-      [:return_value, "_returned_"],
+    SPECIAL_LOCAL_VARS = [
+      [:raised_exception, "_raised"],
+      [:return_value,     "_return"],
```

Comment:
I think it should be `returned` instead of `return` to present the passive form, similar to `raised`.
which means `return_value` should also be changed into `returned_value`.

Reply:
In this timing, not returned yet.